src/extension/commandPaletteHandler.ts

```diff
@@ -175,18 +192,21 @@ export class CommandPaletteHandler {
     public static runExponent(): Q.Promise<void> {
         return this.selectProject()
             .then((project: IReactNativeProject) => {
-                return this.loginToExponent(project)
-                    .then(() => {
-                        return this.executeCommandInContext("runExponent", project.workspaceFolder, () => {
-                            const platform = <ExponentPlatform>this.createPlatform(project, "exponent", ExponentPlatform);
-                            return platform.beforeStartPackager()
-                                .then(() => {
-                                    return platform.startPackager();
-                                })
-                                .then(() => {
-                                    return platform.runApp();
+                return CommandPaletteHandler.checkReactNativePackageExistence(project.workspaceFolder.uri.path)
```

Comment:
Please make sure that this check is required here because Expo may have a non-standard setup.

Reply:
Rolled back this change.

Reply:
I tried to run `Debug in Exponent` scenario without `react-native` package in `node_modules` folder and got an error. So I consider that `react-native` package verification function is required here.